packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -255,37 +261,58 @@ class VifAllowedIps extends BaseComponent {
         )}
         <Row>
           <Col size={10}>
-            {showNewIpForm ? (
-              <span onBlur={this._toggleNewIp}>
+            {showIpSelector && (
+              <span onBlur={this._hideIpInputs}>
                 {resourceSet ? (
                   <SelectResourceSetIp
                     autoFocus
                     containerPredicate={this._getIsNetworkAllowed()}
                     onChange={this._addIp}
                     predicate={this._getIpPredicate()}
                     resourceSetId={resourceSet}
+                    value={null}
                   />
                 ) : (
                   <SelectIp
                     autoFocus
                     containerPredicate={this._getIsNetworkAllowed()}
                     onChange={this._addIp}
                     predicate={this._getIpPredicate()}
+                    value={null}
```

Comment:
This fix can be done in another PR.

Reply:
What part of the change exactly?

Reply:
It makes sens according to the comment https://github.com/vatesfr/xen-orchestra/pull/5367#discussion_r526197058